src/hotspot/share/memory/virtualspace.hpp

```diff
@@ -133,6 +133,9 @@ ReservedSpace ReservedSpace::partition(size_t offset, size_t partition_size)
 // Class encapsulating behavior specific of memory space reserved for Java heap.
 class ReservedHeapSpace : public ReservedSpace {
  private:
+
+  // Compressed oop support is not relevant in 32bit builds.
+#ifdef _LP64
```

Comment:
```suggestion

  // Compressed oop support is not relevant in 32bit builds.
#ifdef _LP64

```
The `#endif // _LP64 below have blank lines before and after, so it seems more consistent to also have it here.`

Reply:
Could you indicate if you saw this comment?

Reply:
Oops, missed that.  Fixed.